tests/e2e/gc_rwx_operation_storm.go

```diff
@@ -488,6 +489,9 @@ var _ = ginkgo.Describe("[rwm-csi-tkg] File Volume Operation storm Test", func()
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}
 
+		// Adding an explicit wait time for the recounciler to refresh the status.
+		time.Sleep(120 * time.Second)
```

Comment:
why 2 mins? can you add polling here instead here?

Reply:
fixed it